### PR TITLE
Allow usage of MPSolver.SetStartingLPBasis in Python API 

### PR DESCRIPTION
**Pull request by pet-mit**

### CODE REVIEW

File: ortools/linear_solver/python/linear_solver.i [hunk 1]
```diff
@@ -168,6 +168,32 @@ from ortools.linear_solver.python.linear_solver_natural_api import VariableExpr
...
+  /// basis_status() function of a MPVariable or a MPConstraint (int between
+  /// 0 and 4: FREE = 0, AT_LOWER_BOUND = 1, AT_UPPER_BOUND = 2, FIXED_VALUE = 3,
+  //  BASIC = 4)
+  ///
+  /// WARNING: With Glop, you should disable presolve when using this because
+  /// this information will not be modified in sync with the presolve and will
+  /// likely not mean much on the presolved problem.
+  void SetStartingLpBasis(
+      const std::vector<int>& variable_statuses,
+      const std::vector<int>& constraint_statuses) {
```
[the remark this answers is not in the document]
Comment (pet-mit) [reply ortools/linear_solver/python/linear_solver.i:185]: actually C++ enums are automatically mapped to integers in python. you can test this with ortools for python, for example with the result status enum.
going back from python to C++ : the python enum (actually an integer) cannot be converted implicitely to C++ enum

**Comment by pet-mit** [comment]
@flomnes do you prefer that I open this PR into google's main branch directly instead, to have their opinion?

**Comment by flomnes** [comment]
> @flomnes do you prefer that I open this PR into google's main branch directly instead, to have their opinion?

Yes, I think it's a good idea

**Comment by pet-mit** [comment]
> > @flomnes do you prefer that I open this PR into google's main branch directly instead, to have their opinion?
> 
> Yes, I think it's a good idea

OK, I opened a PR from our main to theirs first

**Comment by pet-mit** [comment]
see: https://github.com/google/or-tools/pull/4188

**Pull request closed without merging**